charts/linstor-scheduler-extender/templates/deployment.yaml

```diff
@@ -0,0 +1,129 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "linstor-scheduler-extender.fullname" . }}
```

Comment:
The resulting pod name was:

```
linstor-scheduler-linstor-scheduler-extender-7686ddcfdf-bxtqn
```

It's too long in my opinion and does not reflect the current situation.

`linstor-scheduler-extender` name means that there is running just **linstor-scheduler-extender**, but there is actually **kube-scheduler** + **linstor-scheduler-extender**.

How about renaming whole chart to `linstor-scheduler`?